znai-docs/znai/flow/lookup-paths.md

```diff
@@ -15,6 +15,17 @@ When you refer to a file using plugins like
 ../module/src/main/java
 ```
 
+# Class Path Lookup
+
+Znai is written using Java and can access resources in class path. 
```

Comment:
access resources "from the classpath"

Reply:
Could you please suggest the actual change so I can accept it?

Reply:
Even though I'm a member, I don't think I have that permission.